packages/react/src/chakra-provider.tsx

```diff
@@ -1,100 +1,8 @@
-import CSSReset from "@chakra-ui/css-reset"
-import { PortalManager } from "@chakra-ui/portal"
-import {
-  ColorModeProvider,
-  ColorModeProviderProps,
-  GlobalStyle,
-  ThemeProvider,
-  ThemeProviderProps,
-} from "@chakra-ui/system"
+import { ChakraProvider as BaseChakraProvider } from "@chakra-ui/provider"
 import defaultTheme from "@chakra-ui/theme"
-import { Dict } from "@chakra-ui/utils"
-import {
-  EnvironmentProvider,
-  EnvironmentProviderProps,
-} from "@chakra-ui/react-env"
-import * as React from "react"
-import { IdProvider } from "@chakra-ui/hooks"
 
-export interface ChakraProviderProps
```

Comment:
I just experienced a 'breaking change' cuz i cant import `ChakraProviderProps` from `@chakra-ui/react` anymore.
i had to pull from the  `@chakra-ui/provider` pkg.
If that's intended, or desired I'm ok with it, else, it needs to be re-exported here.